<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConverterDialog</class>
 <widget class="QDialog" name="ConverterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>685</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>650</y>
     <width>1081</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="1" column="0">
     <widget class="QLabel" name="selectedSongLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;No audio loaded&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <widget class="QProgressBar" name="progressBar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTipDuration">
       <number>-1</number>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
      <property name="format">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1091</width>
     <height>641</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QGroupBox" name="dataGroupBox">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>signal</string>
        </property>
        <widget class="QCustomPlot" name="customPlotTime" native="true">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>1051</width>
           <height>271</height>
          </rect>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QGroupBox" name="settingsGroupBox">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>settings</string>
        </property>
        <widget class="QGroupBox" name="extractGroupBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>190</y>
           <width>361</width>
           <height>111</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>extract options</string>
         </property>
         <widget class="QComboBox" name="fourierSelectComboBox">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>30</y>
            <width>201</width>
            <height>21</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>Fast Fourier Transform</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Short Time Fourier Transform</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Aquila Short Time Fourier Transform</string>
           </property>
          </item>
         </widget>
         <widget class="QCheckBox" name="variableExtractionCheckBox">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>30</y>
            <width>111</width>
            <height>21</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Variable Extraction</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="extractNumberSpinBox">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>70</y>
            <width>201</width>
            <height>21</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="fileLoadButton">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>60</y>
           <width>91</width>
           <height>23</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Load Audio file</string>
         </property>
        </widget>
        <widget class="QPushButton" name="convertButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>970</x>
           <y>60</y>
           <width>91</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Convert to MIDI</string>
         </property>
        </widget>
        <widget class="QPushButton" name="plotButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>970</x>
           <y>100</y>
           <width>91</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Plot</string>
         </property>
        </widget>
        <widget class="QPushButton" name="backButton">
         <property name="geometry">
          <rect>
           <x>970</x>
           <y>260</y>
           <width>91</width>
           <height>23</height>
          </rect>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
